flight/Modules/CameraStab/camerastab.c

```diff
@@ -376,22 +377,18 @@ static void tablet_info_process() {
 
 		PoiLocationData poi;
 
-		float lat, alt;
-		lat = homeLocation.Latitude / 10.0e6f * DEG2RAD;
-		alt = homeLocation.Altitude;
+		float linearized_conversion_factor_f[3];
+		LLA2NED_linearization_float(homeLocation.Latitude, homeLocation.Altitude, linearized_conversion_factor_f);
```

Comment:
In a future PR, this should be moved out of the camera stab loop. No sense in recalculating it every time.